docs/repositories.md

```diff
@@ -232,6 +232,12 @@ poetry source add --priority=supplemental https://foo.bar/simple/
 
 There can be more than one supplemental package source.
 
+{{% warning %}}
+
+Take into account that someone could publish a new package to PyPI which matches one in your supplemental source.
```

Comment:
```suggestion
Take into account that someone could publish a new package to a primary source which matches one in your supplemental source.
```

I'd put it more generally.

Reply:
Okay! How about this?

Reply:
I think the second sentence is a valuable addition. 👍